multimodal/src/autogluon/multimodal/models/mmdet_image.py

```diff
@@ -390,6 +391,18 @@ def get_layer_ids(
 
         return name_to_id
 
+    def get_backbone_layer_names(self):
+        backbone_layer_names = []
+        backbone_layers_patterns = [
+            "backbone",
+            "encoder",
+        ]
+        for n, _ in self.named_parameters():
+            for pattern in backbone_layers_patterns:
```

Comment:
What if the layer name has both "backbone" and "encoder" in it? The layer name will be added twice in current impl? Is that intended?

Reply:
Thanks for pointing this out. Made the edits to use `any()`.

Ideally "model.backbone.xxx" parameter name is used by non transformer models and "model.encoder.xxx" is used by transformer based models. So there should be no conflicts. Also it won't cause a problem even if there is.